account_invoice_edifact/models/account_move.py

```diff
@@ -0,0 +1,297 @@
+# Copyright 2023 Camtocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime
+
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def edifact_export_invoice(self):
```

Comment:
It's just generating data, not really exporting

```suggestion
    def edifact_invoice_generate_data(self):
```